packages/nuxt/src/app/plugins/check-if-layout-used.ts

```diff
@@ -16,7 +17,10 @@ export default defineNuxtPlugin({
       }
     }
     if (import.meta.server) {
-      nuxtApp.hook('app:rendered', checkIfLayoutUsed)
+      nuxtApp.hook("app:rendered", async () => {
+        await nextTick()
+        checkIfLayoutUsed()
+      });
```

Comment:
seems like there might be something wrong with our eslint setup 🤔 

```suggestion
      nuxtApp.hook('app:rendered', async () => {
        await nextTick()
        checkIfLayoutUsed()
      })
```